libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -58,6 +58,11 @@ static const uint8_t streamSequence[] = { 0 }; // List of 0 based stream Ids tha
 
 static const uint8_t numStreamSequenceIndexes = sizeof(streamSequence)/sizeof(streamSequence[0]);
 
+static uint32_t        last_init_time_ms                 = 0;
+static const uint16_t  MIN_WAIT_TIME_BEFORE_REINIT_MS    = 500;
+static const uint8_t   MAX_VERSION_STRING_RETRY_ATTEMPTS = 4;
+static const uint8_t   MAX_READ_ERRORS_THRESHOLD         = 5;
```

Comment:
Could you please add a comment about the reasoning for selecting these limits? e.g. why is the max read errors threshold 5 rather than 6?

Reply:
Done!